src/main/java/sg/ncl/MainController.java

```diff
@@ -1,5 +1,6 @@
 package sg.ncl;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.fasterxml.jackson.core.JsonProcessingException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)